packages/@n8n/api-types/src/dto/invitation/invite-users-request.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from 'zod';
+
+const roleSchema = z.enum(['global:member', 'global:admin']);
+
+const invitedUserSchema = z.object({
+	email: z.string().email(),
+	role: roleSchema.default('global:member'),
+});
+
+const invitationsSchema = z.array(invitedUserSchema);
+
+export class InviteUsersRequestDto extends Array<z.infer<typeof invitedUserSchema>> {
```

Comment:
out of curiosity why this is one not a class like the other ones, and needs this static method? 🤔

Reply:
because it's an array, and `Z.class` only works for objects. I did not want to wrap this array in an object because this type is also used by a public api, where changing this would be a breaking change.